src/payment/PaymentForm.test.jsx

```diff
@@ -174,14 +176,29 @@ describe('<PaymentForm />', () => {
       ));
       const paymentForm = wrapper.find(PaymentFormComponent).first().instance();
       const values = {
-        valid: 'Good!',
-        invalid: '',
+        firsName: 'Jane',
+        lastName: undefined,
       };
       const expectedErrors = {
-        invalid: 'This field is required',
+        lastName: 'This field is required',
       };
-
       expect(paymentForm.validateRequiredFields(values)).toEqual(expectedErrors);
     });
   });
+  describe('scrollToError', () => {
+    it('scrolls to the input name of the first error', () => {
+      const wrapper = mount((
```

Comment:
Consider moving the setup of `paymentForm` to a setup function like beforeEach(), or some other shared function.  See https://jestjs.io/docs/en/setup-teardown.html

Reply:
Great, added `let wrapper;
    beforeEach(() => {
      wrapper = mount((
        <IntlProvider locale="en">
          <Provider store={mockStore(storeMocks.defaultState)}>
            <PaymentForm handleSubmit={() => {}} />
          </Provider>
        </IntlProvider>
      ));
    });`